app/controllers/hubstats/pull_requests_controller.rb

```diff
@@ -33,6 +33,7 @@ def show
       @pull_request = Hubstats::PullRequest.belonging_to_repo(@repo.id).where(id: params[:id]).first
       @comments = Hubstats::Comment.belonging_to_pull_request(params[:id]).includes(:user).created_since(@timespan).limit(20)
       @comment_count = Hubstats::Comment.belonging_to_pull_request(params[:id]).includes(:user).created_since(@timespan).count(:all)
+      @deploys = Hubstats::Deploy.where(id: @pull_request.deploy_id)
```

Comment:
I'm reusing a view that is also used when showing multiple deploys, so that's why there's a length check. And it still doesn't like the workaround, but instead likes to complain about users...